pandunia-lekse.csv

```diff
@@ -377,6 +388,9 @@ bujete|||eng:fra:budget, deu:Budget, rus:бюджет (b’udzhet), tur:bütçe,
 buke|||eng:book, deu:Buch, may:buku, swa:buku, fra:bouquin, hin:बुक (buk)|book|livre|libro|livro|книга||书|本 (書物)|||पुस्तक (किताब)||buku (pustaka, kitab)||kitap|libro|kirja|książka
 bula||||boil (cook)||hervir (cocer)|||||||||||||boligi|keittää (kiehuttaa)|zapomnieć, zapominać
 bulbule|||may:bulbul, tur:bülbül, hin:बुलबुल (bulbul), urd:(bulbul), fas:(bolbol)|nightingale||ruiseñor|||||||||||||najtingalo|satakieli|słowik
+bule|||eng:bubble, hun:buborék, hin:बुल्ला (bullā), tgl:bula, fra:bulle, pnb:ਬੁਲਬੁਲਾ (bulabulā)|bubble||burbuja (pompa)|||||||||||||bobelo|kupla|bańka
```

Comment:
This conflicts with "bula" (to boil).

Reply:
Wait, did _I_ propose "bule"? I thought that one was already there! Maybe I misread "boil" as "bubble".

Well the main reason I needed it was to make a compound for "tuber" that sounded vaguely like "tuber" and "ubi". In lieu of that, perhaps the most straightforward compound would instead be "masmule". And then "lilbole" for "bubble" works fine.